universal-login-provider/src/ULIFrameProvider.ts

```diff
@@ -114,7 +135,20 @@ export class ULIFrameProvider {
 
     element.innerHTML = `
       <img src="${this.config.logoUrl}" alt="U" >
+      <div id='uni-notifications'></div>
     `;
+    const notificationIndicator = element.getElementsByTagName('div')[0];
```

Comment:
Adding something like this here should simplify the code above
```typescript
this.isNewNotifications.pipe(forEach(hasNotifications => { notificationIndicator.style.display = hasNotifications ? 'block' : 'none' }))
```
This would also make `setNotificationsIndicator` method unneeded